src/main/test/com/epam/esm/dao/GiftCertificateDaoTest.java

```diff
@@ -0,0 +1,107 @@
+package com.epam.esm.dao;
```

Comment:
By maven's convention tests should be located in a folder `root_folder\src\test\java\com\epam\...`.